src/init/application/application.ts

```diff
@@ -58,13 +59,11 @@ export function createApplicationGenerator(templateName: string):
           message: `Main element name`,
           default: (answers: any) => `${answers.name}-app`,
           validate: (name: string) => {
-            const nameContainsHyphen = name.includes('-');
-            if (!nameContainsHyphen) {
-              this.log(
-                  '\nUh oh, custom elements must include a hyphen in ' +
-                  'their name. Please try again.');
+            const validation = validateElementName(name);
+            if(!validation.isValid) {
+              this.log(`\nUh oh. ${validation.message} Please try again.`);
```

Comment:
I think you can remove the "Uh oh" part and put "Please try again" on a new line.